Script error - return [ERROR] UAM (retry: 0) -> clearance cookie missing! Unknown error -> CF clearance could not be found

Hi,

i use example with this page: http://alltube.tv/

but when i execute code i get error like in subject.
Try to set cookie, delay - nothing work.

Debug data:
`[INFO] UAM (retry: 0) -> waiting for 10 seconds! [INFO] UAM (retry: 0) -> getting form params... [INFO] UAM (retry: 0) -> (r param: 5fb9d377a4ffbc8455c260fe51b6b152ccf599f1-1575320911-0-AZw+597QLuV9Hm+m0HeFaE9ZpcmAAx3swrdjPq5tKZ9OWUL5kJysrObVgMFlwEs5OO3uzTnef7O/aptqe+Mcy9/arSqsc0+5RFBYW5PWWM5go31hNDsfuGIwruZmHCsytT3h2qcjncAw3qhn0O891/EtZQ02sy/2GWsuX8bZx3i91m2a+fqXfmH+tbZWW4nr+Pvz11OGvs4Z2vXuq8VqF29ktgnjOhEjvBLXnVZj1VnzQyBXGqgQWFRPpdPKFOiBP6wgoLZfKZr4v9z8/YFoDTapyIH9o8MDmt0zR46y23wBS2T5TcOsSWY7PKUfowvCbtsxlydSdnGnBauEPCr4en2AzzYQNA9bDhF255HYu0a9mK1P5MSj5tkxwu9Po12tWgpZN/h6G8DWUAmpLEQxK1+dc9s1zZpoDtwxeaY2NTyJQ+blMxNo268aXzZwUMQvnWNMoLx9bkUvXzjodGyE0P6a7S63ZuHSURyHR4PG4YJ+SoL9zHr+CZDOBwrqZzosey4XrCkTtM+uTbrDRAACOAlX/bw5xyMwT41JyQGfOufH7gaMsYrlF+lk+3k/cIGnrDKnBYOkv/5YK8zzUxHNlS1C4i7Dm4AD5d5GaIva04FJKeRXL+MT8VIlzoLz0H1ID68u8Hc9Tu8AJgr0KFHGmYdjhfWN6shV8VtTBqsJkz7x) [INFO] UAM (retry: 0) -> (jschl_vc param: 88b25e9259e36566a89d6c5d9d258dec) [INFO] UAM (retry: 0) -> (pass param: 1575320915.637-+CkVlya47p) [INFO] UAM (retry: 0) -> (jschl_answer param: 15.9295684833) [INFO] Array ( [0] => #HttpOnly_.alltube.tv TRUE / FALSE 1577912911 __cfduid dbcd90197c4cfb00e595c8dc5689a8eac1575320911 ) [ERROR] UAM (retry: 0) -> clearance cookie missing! Unknown error -> CF clearance could not be found!`

I get last version from repo.

Any tips?

--- a/.github/ISSUE_TEMPLATE.md
+++ b/.github/ISSUE_TEMPLATE.md
@@ -1,0 +1,52 @@
+<!--
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. You can ensure this by searching the issue list for this
+repository. If there is a duplicate, please close your issue and add a comment
+to the existing issue instead.
+
+If you suspect your issue is a bug, please edit your issue description to
+include the BUG REPORT INFORMATION shown below. If you fail to provide this
+information, we cannot debug your issue and will close it. We will, however,
+reopen it if you later provide the information.
+
+You do NOT have to include this information if this is a FEATURE REQUEST
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
+**Steps to reproduce the issue:**
+1.
+1.
+1.
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Additional information you deem important (e.g. issue happens only occasionally):**
+
+**Provide some logs**
+
+<details>
+<pre>
+(paste logs)
+</pre>
+</details>
+
+**Version you are using (you may find this information in your `composer.lock` file)**
+
+```
+(paste your output here)
+```
+
+**PHP Version you are using**
+
+```
+(paste your output here)
+```

